### PR TITLE
Renamed roles and added patreon info

### PR DESCRIPTION
**Pull request by ryuk201198**
1. Renamed Assassin back to Evil Guesser
2. Renamed Vigilante back to Nice Guesser
3. Renamed Jailor to Jailer (for better english)
4. added mising patreon info

### CODE REVIEW

**Comment by KingJacob105** [comment]
YESSSSS

File: Modules/DevManager.cs
Comment (0xDrMoe) [inline Modules/DevManager.cs:1]: I'm pushing a change on this, can you pull and PR _before_ I actually commit this?

File: Resources/Data/en.json
Comment (0xDrMoe) [inline Resources/Data/en.json:1]: Same with this, pls

**Review by 0xDrMoe — CHANGES_REQUESTED** [review]

File: Modules/DevManager.cs
Comment (ryuk201198) [reply Modules/DevManager.cs:1]: alr

**Comment by 0xDrMoe** [comment]
have a look now

**Comment by 0xDrMoe** [comment]
double check your changes are confirmed within the PR

**Review by 0xDrMoe — APPROVED** [review]

**Comment by Ultradragon005** [comment]
this was CRAZY ya'll

**Pull request merged**